web/components/DayBlock.jsx

```diff
@@ -1,12 +1,61 @@
 import React from 'react';
+import Flex from './Flex';
+import Text from './Text';
+import Card from './Card';
 
 export default class DayBlock extends React.Component {
+
+    renderTinySetBlocks = (setBlocks) => {
+        return setBlocks.map( setBlock => (
+            <Card
+                key={setBlock.id}
+                height='8px'
+                width='5px'
+                borderBottom={setBlock.blockFraction === '1.0' ? '' : '4px lightGrey solid'}
+                bg='red'
+                my='0.3rem'
+                mr='0.3rem'
+            >
+            </Card>
+        ))
+    }
+
     render() {
+        const { day, selected, onClick } = this.props
+
+        // TODO Remove this mock data and use a graphQl api
+        const setBlocks = [{ id: '1', blockFraction: '1.0' }, { id: '2', blockFraction: '0.5' }, { id: '3', blockFraction: '1' }, { id: '4', blockFraction: '-0.5' }];
+
         return (
-            <div className='DayBlock'>
-                <h6>DayBlock</h6>
-            </div>
+            <Flex
+                column
+                center
+                className='DayBlock'
+                mx='0.5rem'
+            >
+                <Card
+                    width='100%'
+                    bg='white'
+                    borderLeft={selected ? '2px solid red' : '2px solid white'}
+                    depth={9}
+                    mx='0.5rem'
+                    onClick={() => onClick(day)}
```

Comment:
I saw this and started to analyze the code to figure out what is supposed to happen when a DayBlock is clicked.

I think to minimize the number of files we have to jump around and look at, we can use redux to call a state change closer to this component. Currently we have a state change of `goToScheduleDay` in the `SchedulePage` component, which is passed down to the `SideBar` which is passed down to the `DayBlock`. It is useful to keep the `onClick` abstracted away from the bottom-most component (DayBlock), but I think moving the state change call to the `SideBar` component would be a little bit cleaner.

This will mean refactoring the state structures to use Redux so we can access them from any component we choose.